report/renderer/category-renderer.js

```diff
@@ -91,12 +91,12 @@ export class CategoryRenderer {
         packElmImg.classList.add('lh-audit__stackpack__img');
         packElmImg.src = pack.iconDataURL;
         packElmImg.alt = pack.title;
-        packElm.appendChild(packElmImg);
+        packElm.append(packElmImg);
```

Comment:
append can take multiple params.. so in this case you can combine L96 into this one. 

(probably makes sense to assign `this.dom.convertMarkdownLinkSnippets(pack.description)` to a var for clarity to and avoiding line-length issues)

Reply:
Updated in latest. Resolving.